src/test/java/uk/gov/hmcts/ccd/domain/service/processor/GlobalSearchProcessorServiceTest.java

```diff
@@ -649,4 +739,14 @@ private void assertSearchCriteriaFields(Map<String, String> expectedValues) {
 
     }
 
+    private void assertSearchCriteriaFieldIsNull() {
```

Comment:
Rename as the check is verifying the SearchPartyNode is null.  

(NB: the complete code should never return a null SearchCrtieria when GlobalSearch is enabled but an empty one is ok)

Reply:
done